examples/ruby/rideshare_rails/config/secrets.yml

```diff
@@ -0,0 +1,2 @@
+production:
+  secret_key_base: placeholder
```

Comment:
Is this actually needed for the example Rails app?

Reply:
Yeah, I think so